src/components/VmDetails/cards/NicsCard/NicEditor.js

```diff
@@ -0,0 +1,302 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import {
+  Button,
+  Col,
+  ControlLabel,
+  FieldLevelHelp,
+  Form,
+  FormControl,
+  FormGroup,
+  Modal,
+  Radio,
+} from 'patternfly-react'
+import SelectBox from '../../../SelectBox'
+import ExpandCollapseSection from '../../../ExpandCollapseSection'
+import NicLinkStateIcon from './NicLinkStateIcon'
+
+import { msg, enumMsg } from '../../../../intl'
+import style from './style.css'
+
+const EMPTY_ID = ''
+
+const NIC_INTERFACES = [
+  {
+    id: 'rtl8139',
+    value: enumMsg('NicInterface', 'rtl8139'),
+  },
+  {
+    id: 'e1000',
+    value: enumMsg('NicInterface', 'e1000'),
+  },
+  {
+    id: 'virtio',
+    value: enumMsg('NicInterface', 'virtio'),
+  },
+]
+const NIC_INTERFACE_DEFAULT = 'virtio'
+const NIC_INTERFACE_CANT_CHANGE = [ 'pci_passthrough' ]
+
+const LabelCol = ({ children, ...rest }) => {
```

Comment:
Actually we use `...props` elsewhere, so for consistenty, please, rename `...rest` to `...props`